test/unit/dialects/mssql/query-generator.test.js

```diff
@@ -119,12 +119,16 @@ if (current.dialect.name === 'mssql') {
         sequelize: this.sequelize,
         dialect: this.sequelize.dialect,
       });
+
+      let dbVersion = '11.0.0';
+
       // Test newer versions first
       // Should be all the same since handling is done in addLimitAndOffset
       // for SQL Server 2012 and higher (>= v11.0.0)
       modifiedGen.sequelize = {
+        getDatabaseVersion: () => dbVersion,
```

Comment:
this change shouldn't be necessary. getDatabaseVersion already returns options.databaseVersion